packages/gasket-plugin-analyze/EXAMPLES.md

```diff
@@ -0,0 +1,142 @@
+# @gasket/plugin-analyze Examples
+
+This document provides working examples for using the `@gasket/plugin-analyze` package.
+
+## Plugin Usage
+
+### Basic Plugin Installation
+
+```js
+// gasket.js
+import { makeGasket } from '@gasket/core';
+import pluginAnalyze from '@gasket/plugin-analyze';
+
+export default makeGasket({
+  plugins: [
+    pluginAnalyze
+  ]
+});
+```
+
+### Configuration Interface
+
+The plugin extends the `GasketConfig` interface with `bundleAnalyzerConfig`:
+
+```js
+// gasket.js
+import { makeGasket } from '@gasket/core';
+import pluginAnalyze from '@gasket/plugin-analyze';
+
+export default makeGasket({
+  plugins: [
+    pluginAnalyze
+  ],
+  bundleAnalyzerConfig: {
+    browser: {
+      analyzerMode: 'static',
+      reportFilename: 'custom-browser-report.html',
+      defaultSizes: 'gzip',
+      openAnalyzer: false
+    },
+    server: {
+      analyzerMode: 'static',
+      reportFilename: 'custom-server-report.html',
+      openAnalyzer: false
+    }
+  }
+});
+```
+
+### Environment-Specific Configuration
+
+```js
+// gasket.js
+import { makeGasket } from '@gasket/core';
+import pluginAnalyze from '@gasket/plugin-analyze';
+
+export default makeGasket({
+  plugins: [
+    pluginAnalyze
+  ],
+  environments: {
+    'local.analyze': {
+      bundleAnalyzerConfig: {
+        browser: {
+          analyzerMode: 'server',
+          analyzerPort: 8888,
+          openAnalyzer: true
+        },
+        server: {
+          analyzerMode: 'server',
+          analyzerPort: 8889,
+          openAnalyzer: true
+        }
+      }
+    }
+  }
+});
+```
+
+## Runtime Usage
+
+### Running Analysis via Environment Variable
```

Comment:
Add a note that analyze will not run unless the `ANALYZE` env var is set, or the `.analyze` sub-environment is used.